conans/cli/commands/create.py

```diff
@@ -14,7 +15,17 @@
 from conans.util.files import chdir, mkdir
 
 
-@conan_command(group=COMMAND_GROUPS['creator'])
+def json_create(info):
+    deps_graph, profile_build, profile_host = info
+    result = {
+        "profile_build": profile_build.serialize(),
+        "profile_host": profile_host.serialize(),
+        "deps_graph": deps_graph.serialize(),
```

Comment:
lets use just ``graph`` for naming
```suggestion
        "graph": deps_graph.serialize(),
```